packages/core/__tests__/language-server/diagnostic-augmentation.test.ts

```diff
@@ -583,7 +583,24 @@ describe('Language Server: Diagnostic Augmentation', () => {
               "line": 10,
             },
           },
-          "relatedInformation": [],
+          "relatedInformation": [
+            {
+              "location": {
+                "range": {
+                  "end": {
+                    "character": 83,
+                    "line": 18,
+                  },
+                  "start": {
+                    "character": 69,
+                    "line": 18,
+                  },
+                },
+                "uri": "file:///PATH_TO_MODULE/@glint/template/-private/integration.d.ts",
+              },
+              "message": "'[InvokeDirect]' is declared here.",
+            },
+          ],
```

Comment:
This additional snapshotted data just means that Volar has since enhanced the metadata provided on each diagnostic, e.g. you should start to see more "SYMBOL is declared here" messages with links to the declaration source.